DFC.Digital/DFC.Digital.Web.Sitefinity.WebApi/Mvc/Controllers/AVFeedRecycleBinController.cs

```diff
@@ -0,0 +1,26 @@
+﻿using DFC.Digital.Data.Interfaces;
+using System.Net;
+using System.Web.Http;
+using Telerik.Sitefinity.Utilities.MS.ServiceModel.Web;
+using Telerik.Sitefinity.Web.Services;
+
+namespace DFC.Digital.Web.Sitefinity.WebApi.Mvc.Controllers
+{
+    public class AVFeedRecycleBinController : ApiController
+    {
+        private readonly IRecycleBinRepository recyleBinRepository;
+
+        public AVFeedRecycleBinController(IRecycleBinRepository recyleBinRepository)
+        {
+            this.recyleBinRepository = recyleBinRepository;
+        }
+
+        // DELETE dfcapi/avfeedrecyclebin
+        public IHttpActionResult Delete(int count)
+        {
+            ServiceUtility.RequestBackendUserAuthentication();
```

Comment:
We can use/modify the IWebAppContext.checkAuthentication method so that we can test this controller with Sitefinity dependency